.env.example

```diff
@@ -3,4 +3,5 @@ REACT_APP_GOOGLE_OAUTH_URL=https://accounts.google.com/o/oauth2/v2/auth
 REACT_APP_GOOGLE_CLIENT_ID=test_client
 REACT_APP_DASHBOARD_URL=http://localhost:3000
 REACT_APP_TOKENINFO=https://oauth2.googleapis.com/tokeninfo
+REACT_APP_DEVMODE=false
```

Comment:
The traditional convention for this is `NODE_ENV` with settings of `develop`, `production` or `test`. However, I'd do a fast regression test because CRA might pick that up and do things we don't expect.

Reply:
Research changed my opinion, set it `REACT_APP_DEV_DATA` and we're good. Be sure to update `dashboard.js` too.